tests/nightly/test_dropout_with_seed.py

```diff
@@ -0,0 +1,43 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import mxnet as mx
+import numpy as np
+from mxnet.test_utils import assert_almost_equal
+
+
+def test_dropout_with_seed():
```

Comment:
Add the with seed annotation

Reply:
I'm manually choosing a random seed and set it before each forward, so with_seed decorator will not take effect. See comment: https://github.com/apache/incubator-mxnet/pull/16532#discussion_r336370708

Reply:
In line 29 you're generating a random number to feed the seed though, so that generator needs to be fed with a seed as well